roc/log.go

```diff
@@ -181,6 +192,36 @@ func loggerRoutine() {
 	}
 }
 
+func getGID() uint64 {
+	b := make([]byte, 64)
+	b = b[:runtime.Stack(b, false)]
+	b = bytes.TrimPrefix(b, []byte("goroutine "))
+	b = b[:bytes.IndexByte(b, ' ')]
+	n, _ := strconv.ParseUint(string(b), 10, 64)
+	return n
+}
+
+func logWrite(level LogLevel, text string) {
+	if level >= LogDebug {
+		module, ok := debug.ReadBuildInfo()
```

Comment:
I think we can just hard-code module to "roc_go". It will fit well with other modules used for logging - big internal parts of roc-toolkit, like "roc_core", "roc_audio", "roc_rtp", etc.